routes/conf.js

```diff
@@ -40,57 +60,30 @@ function get_conf () {
 
 
 router.get('/conf', async function(req, res){
-    let terms_of_use = '/doc/terms_of_use.txt';
-    if (CONFIG.general.terms_of_use) {
-        terms_of_use = CONFIG.general.terms_of_use;
-    }
-    let default_home = 'user'; // can be user or project
-    if (CONFIG.general.web_home) {
-        default_home = CONFIG.general.web_home;
-    }
-    let enable_ui = {
-        'messages': true,
-        'databases': true,
-        'tps': true,
-        'websites': true,
-        'u2f_key': true,
-        'ip': true,
-        'newsletters': true,
-        'log': true
-    };
-    if (CONFIG.enable_ui) {
-        enable_ui = CONFIG.enable_ui;
-    }
-    enable_ui.main_group = CONFIG.general.use_group_in_path;
-    enable_ui.user_group = !CONFIG.general.disable_user_group;
-
-    let duration = Object.keys(get_conf().duration);
-
     let config = {
-        'main_groups': CONFIG.general.main_groups,
-        'terms_of_use': terms_of_use,
-        'default_home': default_home,
-        'name': CONFIG.general.name,
-        'support': CONFIG.general.support,
+        'main_groups': get_conf().general.main_groups,
```

Comment:
better to call get_conf once before, then use object
during object dict, multiple get_conf will check for is_init, which may not be set yet.

Reply:
https://github.com/genouest/genouestaccountmanager/pull/152/commits/c7ef5964a0b32052bbeac678d36bf63bee77117a